### ISSUE TITLE
Problem with not showing scrollbar

### ISSUE THREAD

**Issue by nickdima** [moodysalem/react-fixed-page#3]
I'm using the component for blocking the page when the user's hovers inside a dropdown.
There's no problem when testing on a Mac with the trackpad or a magic mouse. But connecting a normal mouse which makes the browser always show the scrollbar track this happens:

![giphy](https://user-images.githubusercontent.com/614468/28015646-24768dea-6571-11e7-97c5-8f83dec9d687.gif)

**Comment by moodysalem** [comment]
Hi Nick. What solution would you propose for this? I think the best implementation would visually indicate that you cannot scroll but without shifting the content or changing how much scrollable area appears to be available.

**Comment by nickdima** [comment]
Yeah, I agree. Probably the biggest problem here is the content shifting. Not sure how it could be resolved.
